selfdrive/car/tests/test_car_interfaces.py

```diff
@@ -18,13 +19,40 @@
 from openpilot.selfdrive.controls.lib.longcontrol import LongControl
 from openpilot.selfdrive.test.fuzzy_generation import FuzzyGenerator
 
+DrawType = Callable[[st.SearchStrategy], Any]
+
 ALL_ECUS = {ecu for ecus in FW_VERSIONS.values() for ecu in ecus.keys()}
 ALL_ECUS |= {ecu for config in FW_QUERY_CONFIGS.values() for ecu in config.extra_ecus}
 
 ALL_REQUESTS = {tuple(r.request) for config in FW_QUERY_CONFIGS.values() for r in config.requests}
 
-MAX_EXAMPLES = int(os.environ.get('MAX_EXAMPLES', '60'))
+MAX_EXAMPLES = int(os.environ.get('MAX_EXAMPLES', '20'))
```

Comment:
This won't affect the test coverage in CI (or the speed of the tests) since we already specify the `MAX_EXAMPLES` in our yaml, but it will if you run this test outside of CI. 

Even though it won't technically reduce coverage, this change kind of goes against the requirements of the issue, since we explicitly specify `"with the same test coverage"`.

Reply:
I can see this change easily reducing test coverage, but I'm disregarding what we currently use in CI.